examples/virtual_specs/construct.yaml

```
@@ -0,0 +1,22 @@
+name: virtual_specs
+
+version: 0.0.1
+
+keep_pkgs: True
+
+channels:
+  - conda-forge
+
+specs:
+  - ca-certificates
+
+virtual_specs:
+  - __osx>=20    # [osx]
+  - __glibc>=20  # [linux]
```

Should we lose `<=` here, too? I know `__glibc` is only version 2, so it's unrealistic it will get to 20, but who knows with `__osx`

---

Sure, do you mean like this?

```suggestion
  - __osx<=1    # [osx]
  - __glibc>=20  # [linux]
```

---

Added